tests/ZendTest/Code/Reflection/TestAsset/TestTraitClass4.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace ZendTest\Code\Reflection\TestAsset;
+
+use ZendTest\Code\Reflection\TestAsset\TestTraitClass3 as TestTrait;
+
+//issue #7428
+class TestTraitClass4
+{
+	use TestTrait;
+
+	/**
+	* @var bool
+	*/
+	protected static $other = false;
+
+
+	/**
+	* Constructor
+	*
+	* @param bool $other
+	*/
+	public function __construct($other)
+	{
+		$this->other = $other;
+	}
```

Comment:
Fixed that. I've also removed unnecessary code from test classes. The only check done on them is presence of traits. Other functions and variables are not needed.